<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8">
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>42</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(0, 0, 0);
}

QPushButton{
background-color: #000000ff;
border-radius: 20px;
}

QPushButton:hover {
background-color: rgb(61, 61, 61);
border-radius: 20px;

}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="bt_menu">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../imagenes/menu.png</normaloff>../../imagenes/menu.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>477</width>
              <height>38</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bt_restaurar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../imagenes/minimizar.png</normaloff>../../imagenes/minimizar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_minimizar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../imagenes/reducir.png</normaloff>../../imagenes/reducir.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_maximizar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>ampliar.png</normaloff>ampliar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_cerrar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>cerrar.png</normaloff>cerrar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_contenido">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_control">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>0</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{ 
background-color: rgb(225, 161, 11);
}

QPushButton{
background-color: rgb(61, 61, 61);
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
color: rgb(255, 255, 255);
font: 77 10pt &quot;Arial Black&quot;;
}

QPushButton:hover {
background-color: white;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
color: rgb(0, 0, 0);
font: 77 10pt &quot;Arial Black&quot;;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="bt_datos">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Consultar Producto</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_agregar_producto">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Agregar Producto</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_actualizar_nombre">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Actualizar Nombre</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_paginas">
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color: rgb(61, 61, 61);
}

QLabel{
font: 87 12pt &quot;Arial Black&quot;;
background-color: #000000ff;
color: rgb(225, 161, 11);
border:0px solid #14C8DC;
}

QLineEdit{
border:0px;
color: rgb(225, 161, 11);
border-bottom: 2px solid rgb(61, 61, 61);
font: 75 12pt &quot;Times New Roman&quot;;
}

QPushButton{
background-color: rgb(61, 61, 61);
border-radius: 15px;
color: rgb(255, 255, 255);
font: 77 10pt &quot;Arial Black&quot;;
}


QPushButton:hover{
background-color: rgb(225, 161, 11);
border-radius: 15px;
color: rgb(0, 0, 0);
font: 77 10pt &quot;Arial Black&quot;;
}

QTableWidget {
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font-size: 8pt;
gridline-color: rgb(0, 206, 151); 
color: #000000;
}

QHeaderView::section {
background-color: rgb(225, 161, 11);
border: 1px solid rgb(0, 0, 0);
font-size: 8pt;
}

QTableWidget QTableComerButton::section {
background-color: rgb(0, 0, 0);

}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <widget class="QWidget" name="page_actualizar">
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>ACTUALIZAR NOMBRE PRODUCTO</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <widget class="QLabel" name="label_11">
                     <property name="text">
                      <string>Id del Producto a Actualizar:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="act_buscar"/>
                   </item>
                   <item>
                    <widget class="QPushButton" name="bt_actualiza_buscar">
                     <property name="minimumSize">
                      <size>
                       <width>120</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>BUSCAR</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QLabel" name="label_12">
                     <property name="text">
                      <string>Nuevo Nombre</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="act_nombre"/>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_6">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="signal_actualizar">
                     <property name="minimumSize">
                      <size>
                       <width>120</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_5">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="bt_actualizar">
                     <property name="minimumSize">
                      <size>
                       <width>120</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>ACTUALIZAR</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_agregar">
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>AGREGAR PRODUCTO</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_7">
                     <property name="spacing">
                      <number>30</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="text">
                        <string>No Identificador</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>Nombre</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>Peso o Volumen</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>Fecha Vencimiento</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>Precio de Produccion</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>Precio de Venta</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_6">
                     <property name="spacing">
                      <number>30</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="reg_id"/>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="reg_nombre"/>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="reg_peso"/>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="reg_vencimiento"/>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="reg_produccion"/>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="reg_venta"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="signal_agregar">
                     <property name="minimumSize">
                      <size>
                       <width>200</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="bt_agregar">
                     <property name="minimumSize">
                      <size>
                       <width>200</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>AGREGAR</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_datos">
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="font">
                    <font>
                     <family>Arial Black</family>
                     <pointsize>12</pointsize>
                     <weight>10</weight>
                     <italic>false</italic>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>PRODUCTOS</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="tabla_productos">
                   <column>
                    <property name="text">
                     <string>noIdProducto</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>nombreProducto</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>pesoVolumen</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>fechaVencimiento</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>precioProduccion</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>precioVenta</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="bt_refrescar">
                     <property name="minimumSize">
                      <size>
                       <width>120</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Refrescar</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
